vite.config.ts

```diff
@@ -1,18 +1,39 @@
 import react from '@vitejs/plugin-react';
+import appRoot from 'app-root-path';
+import path from 'path';
 import { defineConfig as defineViteConfig, mergeConfig } from 'vite';
 import circleDependency from 'vite-plugin-circular-dependency';
 import dts from 'vite-plugin-dts';
 import { defineConfig as defineVitestConfig } from 'vitest/config';
 
 const vitestConfig = defineVitestConfig({
   test: {
-    deps: { interopDefault: true },
-    environment: 'jsdom',
-    css: false,
-    globals: true,
-    setupFiles: '../../vitest-setup.ts',
     watch: false,
+    globals: true,
+    setupFiles: path.join(appRoot.path, 'vitest-setup.ts'),
```

Comment:
hmm, gjerne forklar med kommentar hvorfor dette trengs. Hvorfor kan du ikke bare si:
```
setupFiles: "./vitest-setup.ts",
```
Det funker for meg lokalt?

Reply:
hm, ja, ikke sant. Det tenkte jeg ikke på 😅 

Men vil ikke dette fungere, så slipper du dra inn bibliotek?
```
setupFiles: path.resolve(__dirname, "vitest-setup.ts"),
```